crates/test-utils/src/bin/dump-katana.rs

```diff
@@ -23,11 +24,19 @@ async fn main() {
         .dump_state()
         .expect("Failed to call dump_state on Katana state");
 
-    let state = serde_json::to_string(&dump_state).expect("Failed to serialize state");
-
     // Dump the state
     std::fs::create_dir_all(".katana/").expect("Failed to create Kakata dump dir");
-    std::fs::write(".katana/dump.json", state).expect("Failed to write dump to .katana/dump.json");
+
+    let katana_dump_path = String::from(".katana/dump.json");
+    let katana_dump_file = File::options()
+        .create_new(true)
```

Comment:
i think this will fail if the file already exists, can we check?

Reply:
I have replaced this with [create](https://doc.rust-lang.org/std/fs/struct.File.html#method.create), which should solve this, it should not fail if the file already exists, and since the `append` option is false, it should always overwrite.